crates/forge/tests/fixtures/zk/Create2.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import "forge-std/Test.sol";
+import {L2ContractHelper} from "era-contracts/l2-contracts/contracts/L2ContractHelper.sol"; // =0.8.20
+
+import {Greeter} from "../src/Greeter.sol";
+import {CustomNumber} from "../src/CustomNumber.sol";
+
+import {Create2Utils} from "../src/Create2Utils.sol";
+
+contract Create2Test is Test {
+    function getBytecodeHash(string memory path) internal returns (bytes32 bytecodeHash) {
+        string memory artifact = vm.readFile(path);
+        bytecodeHash = vm.parseJsonBytes32(artifact, ".hash");
+    }
+
+    function testCanDeployViaCreate2() public {
+        bytes32 bytecodeHash = getBytecodeHash("zkout/Greeter.sol/Greeter.json");
+        address sender = address(0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496);
+        bytes32 salt = "12345";
+        bytes32 constructorInputHash = keccak256(abi.encode());
+
+        address expectedAddress =
+            Create2Utils.computeCreate2Address(sender, salt, bytes32(bytecodeHash), constructorInputHash);
+
+        // deploy via create2
+        address actualAddress = address(new ConstantNumber{salt: salt}());
+
+        assertEq(actualAddress, expectedAddress);
+    }
+
+    function testComputeCreate2WithNoArgs() external {
+        bytes32 salt = bytes32(0x0);
+
+        bytes32 bytecodeHash = getBytecodeHash("zkout/Greeter.sol/Greeter.json");
+
+        address computedAddress =
```

Comment:
What's the reason behind the naming of `computed`/`expected`? Why not `expected` for the utils one (as in the other test) and `l2ContractsHelperAddress` for the other one (as you want to test this address as well) ?

Reply:
It was originally a separate test, which only used the util.
But for the reasoning in the other tests, `computed` is due to our code (the util) (which we copied from system contracts), but the source of thruth should be the system contract imo, thus I named the result `expected`.
I'll rename this one to match the rest